src/python/zquantum/core/bitstring_distribution.py

```diff
@@ -208,20 +209,22 @@ def get_qubits_number(self):
 
 
 def compute_clipped_negative_log_likelihood(
-    target_distribution, measured_distribution, epsilon=1e-9
+    target_distribution, measured_distribution, distance_measure_parameters
```

Comment:
I see we don't have typing in this file, but I think this is good time to actually add it :)

Reply:
Added in [this commit](https://github.com/zapatacomputing/z-quantum-core/pull/63/commits/da4ff1b90615db7d3b26384a1a8dd6e2d09d8903)

Reply:
Thank you ❤️

Reply:
I've just seen that in other parts of Orquestra the i/o methods use the `TextIO` type (see [here](https://github.com/zapatacomputing/z-quantum-core/blob/a5f726ca0d2d90145f2300240d76d4c0f0f6be94/src/python/zquantum/core/graph.py#L22)). I've marked it as string and wanted to check what is the best way to go here @mstechly